examples/chart/teleport/templates/deployment.yaml

```diff
@@ -38,7 +38,17 @@ spec:
 {{- if not .Values.proxy.tls.enabled }}
         - --insecure-no-tls
 {{- end }}
+        - --nodename=$(MY_NODE_NAME)
+        - --advertise-ip=$(MY_POD_IP)
```

Comment:
Same here, sometimes for auth servers it's actually better to have the service IP to be advertised (e.g. auth servers0. I see the rationale for having this as an option though, so thinking what's best here.

Reply:
@drewwells @klizhentas I think assigning the advertise IP in this way is fine, as each pod will have a unique IP.